src/auto/log_field.rs

```diff
@@ -0,0 +1,5 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib_sys;
```

Comment:
Empty file

Reply:
Arf, you saw it before I had time to remove it!